apiserver/uniter/uniter.go

```diff
@@ -1685,3 +1685,13 @@ func (u *UniterAPIV3) getOneNetworkConfig(canAccess common.AuthFunc, unitTagArg,
 
 	return results, nil
 }
+
+// SLALevel returns the model's SLA level.
+func (u *UniterAPIV3) SLALevel() (params.StringResult, error) {
```

Comment:
So, we are adding a new API to the uniter facade, but to do this right, the version needs to be bumped. And the api client needs to check what backend version it is talking to and fail gracefully if the agent has not yet been upgraded to the new v4 facade. There's a couple of examples of how to do this floating around in the codebase.